pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -214,7 +215,11 @@ private static void start() {
         boolean doLiquibase = cfg.getBoolean(ConfigFile.DO_LIQUIBASE);
         int maxConnections = cfg.getInt(ConfigFile.NUM_POOLED_CONNECTIONS);
 
-        String firecloudKeysLocation = System.getProperty(ConfigFile.FIRECLOUD_KEYS_DIR_ENV_VAR);
+        // For benefit of GAE. Does not like command line options with "=" characters and env variables with "."
+        String firecloudKeysLocation = System.getenv(ConfigFile.FIRECLOUD_KEYS_DIR_ENV_VAR.replace('.', '_'));
```

Comment:
Not happy about this little transformation here. Wanted to avoid changing every invocation of DataDonationPlatform in maven and all our local environments that use ddp_firecloudKeysDir . And the template files.
A tradeoff between cleanliness and convenience. Feel free to disagree.